CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 # Defines Taraxa library target.
 
+if (DEFINED SYSTEM_HOME_OVERRIDE)
+    list(APPEND CMAKE_PREFIX_PATH ${SYSTEM_HOME_OVERRIDE})
+    include_directories(${SYSTEM_HOME_OVERRIDE}/include)
+    link_directories(${SYSTEM_HOME_OVERRIDE}/lib)
+endif ()
+set(CMAKE_OSX_SYSROOT /)
+set(CMAKE_MACOSX_RPATH ON)
```

Comment:
I'm not completely sure about the effects, but It once helped me solve some problem long ago, and now I just include it everywhere :D